### PR TITLE
Version 0.1.0

### PR DESCRIPTION
**Pull request by wargio**
We still use the original r2 version number.
Let's start with the version 1.0.0-dev

### CODE REVIEW

**Comment by ret2libc** [comment]
I think we said to go with just two numbers for the time being. So 1.0-git.

**Comment by wargio** [comment]
ops, my bad.

**Review by ITAYC0HEN — APPROVED** [review]
Can consider to use team@rizin.re which might be accessible for more people in the future. but it is not critical as we don't expect many emails anyways

**Comment by wargio** [comment]
> Can consider to use [team@rizin.re](mailto:team@rizin.re) which might be accessible for more people in the future. but it is not critical as we don't expect many emails anyways

done

**Review by XVilka — APPROVED** [review]
LGTM. While you are on it - can you please do the same for rz-ghidra, rz-pm, rz-pipe? cc @thestr4ng3r

File: configure [hunk 1]
```diff
@@ -1,8 +1,5 @@
 #!/bin/sh
 # This script was automatically generated by ACR v1.9.4
-# @author: pancake <nopcode.org>
```
Comment (XVilka) [inline configure]: These fields are generated by acr. Please just regenerate this file.

**Review by XVilka — CHANGES_REQUESTED** [review]

File: configure (hunk 1 above)
Comment (wargio) [reply configure]: i do not know who to do that.

**Comment by XVilka** [comment]
See also https://github.com/rizinorg/rizin/issues/174

**Comment by wargio** [comment]
so we want to drop also `-git` ?

**Review by XVilka — APPROVED** [review]

**Review by ret2libc — APPROVED** [review]

**Pull request merged**